app/controllers/shopping_lists_controller.rb

```diff
@@ -0,0 +1,20 @@
+class ShoppingListsController < ApplicationController
+  def index
+    recipe_id = params[:recipe_id]
+    return unless recipe_id.present?
+
+    @recipe = Recipe.find(recipe_id)
+
+    @recipe_foods = @recipe.recipe_foods
+    @total_value = @recipe_foods.sum(&:total_price)
+
+    # @recipe_foods = RecipeFood.all
+    # @total_value = @recipe_foods.sum(&:total_price)
```

Comment:
- [ ] Please make sure to remove the commented code from this file. It's not a professional and good practice to leave commented code in the codebase.